iwf-sdk.yaml

```diff
@@ -70,19 +70,19 @@ paths:
                 $ref: '#/components/schemas/ErrorResponse'
   /api/v1/workflow/dataobjects/get:
```

Comment:
It seems this path needs to be modified after the server supports it?

Reply:
Yeah and we can’t just change the path in the server without more work to support backward compatibility. So I decided to keep the wrong name in the yaml file for now. And it’s invisible for users anyway